code/modules/antagonists/revolution/revolution.dm

```diff
@@ -510,6 +510,8 @@
 	for (var/datum/mind/crewmember as anything in get_crewmember_minds())
 		if (crewmember.has_antag_datum(/datum/antagonist/enemy_of_the_revolution))
 			continue
+		if(crewmember.current && (isobserver(crewmember.current) || crewmember.current.stat == DEAD)) // if we have 60 dead nonrev crew, 2 alive crew, and 10 alive revs, it should qualify for the shuttle
```

Comment:
```suggestion
		if(crewmember.current?.stat == DEAD) // if we have 60 dead nonrev crew, 2 alive crew, and 10 alive revs, it should qualify for the shuttle
```

That being said is it even possible for current to be null here?

Reply:
I don't trust Mind code

Reply:
According to other usages the answer is "maybe" which blows my mind